rbac/api/cross_access/view.py

```diff
@@ -188,36 +172,20 @@ def retrieve(self, request, *args, **kwargs):
         """Retrieve cross account requests by request_id."""
         result = super().retrieve(request=request, args=args, kwargs=kwargs)
 
-        if settings.AUTHENTICATE_WITH_ORG_ID:
-            if validate_and_get_key(self.request.query_params, QUERY_BY_KEY, VALID_QUERY_BY_KEY, ORG_ID) == ORG_ID:
-                user_id = result.data.pop("user_id")
-                principal = PROXY.request_filtered_principals(
-                    [user_id], account=None, org_id=None, options={"query_by": "user_id", "return_id": True}
-                ).get("data")[0]
-
-                # Replace the user_id with user's info
-                result.data.update(
-                    {
-                        "first_name": principal["first_name"],
-                        "last_name": principal["last_name"],
-                        "email": principal["email"],
-                    }
-                )
-        else:
-            if validate_and_get_key(self.request.query_params, QUERY_BY_KEY, VALID_QUERY_BY_KEY, ACCOUNT) == ACCOUNT:
-                user_id = result.data.pop("user_id")
-                principal = PROXY.request_filtered_principals(
-                    [user_id], account=None, org_id=None, options={"query_by": "user_id", "return_id": True}
-                ).get("data")[0]
-
-                # Replace the user_id with user's info
-                result.data.update(
-                    {
-                        "first_name": principal["first_name"],
-                        "last_name": principal["last_name"],
-                        "email": principal["email"],
-                    }
-                )
+        if validate_and_get_key(self.request.query_params, QUERY_BY_KEY, VALID_QUERY_BY_KEY, ORG_ID) == ORG_ID:
+            user_id = result.data.pop("user_id")
+            principal = PROXY.request_filtered_principals(
+                [user_id], account=None, org_id=None, options={"query_by": "user_id", "return_id": True}
```

Comment:
There is no more `account` argument in the `request_filtered_principals` function
Same for the line 196

Reply:
Oh! Good catch! Thanks!